internal/pkg/block/block.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/filecoin-project/go-filecoin/internal/pkg/encoding"
 )
 
+// TODO: If this is gonna stay, it should move to specs-actors
```

Comment:
No, it wouldn't go there because that code won't reference or enforce it. Here is ok for now.